core/controllers/library_test.py

```diff
@@ -433,3 +570,30 @@ def test_cannot_get_private_exploration_summaries_when_logged_out(self):
 
         self.assertEqual(summaries[0]['id'], self.PUBLIC_EXP_ID_EDITOR)
         self.assertEqual(summaries[0]['status'], 'public')
+
+
+class CollectionSummariesHandlerTests(test_utils.GenericTestBase):
+    """Test Collection Summaries Handler."""
+    def test_access_empty_collection(self):
+        response_dict = self.get_json(
+            feconf.COLLECTION_SUMMARIES_DATA_URL,
+            params={'stringified_collection_ids': json.dumps('1')})
+        self.assertDictContainsSubset({
+            'summaries': [],
+        }, response_dict)
+
+    def test_access_non_empty_collection(self):
```

Comment:
Maybe this test can be clubbed with the previous one. First as a pre-check you are testing the summaries to equal []. Then you are creating a collection and the again testing the new value of summaries.

Reply:
fixed